openhands/server/session/session.py

```diff
@@ -99,6 +99,14 @@ async def initialize_agent(
         self.config.security.security_analyzer = (
             settings.security_analyzer or self.config.security.security_analyzer
         )
+        self.config.sandbox.base_container_image = (
+            settings.sandbox_base_container_image
+            or self.config.sandbox.base_container_image
+        )
+        self.config.sandbox.runtime_container_image = (
+            settings.sandbox_runtime_container_image
+            or self.config.sandbox.runtime_container_image
+        )
```

Comment:
If `config.sandbox.runtime_container_image` is already set from another source (such as env. variable `SANDBOX_RUNTIME_CONTAINER_IMAGE` or in config.toml, openhands will use that rather than building the runtime image from user specified `sandbox_base_container_image` which might confuse the user. Maybe instead we should do something like this?
```suggestion
        self.config.sandbox.runtime_container_image = (
            settings.sandbox_runtime_container_image
            if settings.sandbox_base_container_image 
            else self.config.sandbox.runtime_container_image
        )
```